src/components/ui/ImageUpload.tsx

```diff
@@ -40,14 +54,43 @@ class ImageUpload extends Component<Props> {
     field.set('');
   }
 
+  onDropRejected(rejectedFiles) {
```

Comment:
```suggestion
  onDropRejected(rejectedFiles): void {
```

Also, could you add the type of the rejectedFiles, I'm not sure what it is tho